README.md

```diff
@@ -13,18 +13,18 @@ you need to run ```source /opt/ros/melodic/setup.bash``` in your current shell f
 
 ------
 
-On the **first usage**, run ```./setup.sh``` to create virtual environment and install requirements.
+On the **first usage**, run ```sudo ./setup.sh``` to create virtual environment and install requirements.
```

Comment:
I'm usually confused to run `sudo` on some "black box". Another point is that it will be applied for all commands in `./setup.sh` script, whereas it is necessary only in apt commands. For example, `venv` directory will have root access on writing, you will need to apply `sudo` to remove it -- strange behaviour for user space.